DepositRegistrar.sol

```diff
@@ -0,0 +1,306 @@
+// Name registrar using deposits
+
+contract Deed {
+    // The Deed is a contract intended simply to hold ether
+    // It can be controlled only by the registrar and can only send ether back to the owner
+    address public owner;
+    address public bidder;
+    Registrar public registrar;
+    bytes32 public sealedBid;
+    bytes32 public hashedName;
```

Comment:
Well then I'm using name to refer to what you call a second level label: foo.eth.
I could rename it all to label here for consistency, but are you sure this is the proper terminology? I don't think they are very self explanatory. Maybe if we all used "domains"?